homeassistant/components/unifiprotect/services.yaml

```diff
@@ -0,0 +1,34 @@
+set_doorbell_message:
+  name: Set Doorbell message
+  description: >
+    Use to dynamically set the message on a Doorbell LCD screen. Should only be used to set dynamic messages
+    (i.e. setting the current outdoor temperature on your Doorbell). Static messages should still using the Select entity and the
+    add_doorbell_text / remove_doorbell_text services.
+  fields:
+    entity_id:
+      name: Doorbell Text
+      description: (Required) Doorbell to display message on
+      example: "select.front_doorbell_camera_doorbell_text"
+      required: true
+      selector:
+        entity:
+          integration: unifiprotect
+          domain: select
+          device_class: unifiprotect__lcd_message
+    message:
```

Comment:
Message services are usually a notify platform. Our notify platform support is still lacking though so I think it's ok to keep it as a custom service for now.

Reply:
I understand the sentiment to want to get people to use `notify`, but this is not really a notify platform. Changing the text is more like "delivery instructions" rather than a notification. It is literally basically just digital signage with a message on it. You can make the message expire, but the main use case for this service is to set a "dynamic" status message (like you want to display the current temperate).